rfcs/shared/build-system/triage-automation.md

```diff
@@ -0,0 +1,161 @@
+
+Tips for writing a successful RFC:
+
+- Simple plain words that make your point, fancy words obfuscate
+- Try to stay concise, but don't gloss over important details
+- Try to write a neutral problem statement, not one that motivates your desired solution
+- Remember, "Writing is thinking". It's natural to realize new ideas while writing your proposal
+-->
+
+_contributors:_ _@hotell_
+
+---
+
+## Summary
+
+We wanna make initial triage-ing issues a fully automated process.
+
+## Problem statement
+
+At the moment processing newly created issues is mostly manual work done by 2 people on shield duty. While this was a good start it wont scale as the codebase/team/contributors list grows. Thus we need to introduce automation to establish best possible contributors DX while they need our help.
+
+<!--
+Why are we making this change? What problem are we solving? What do we expect to gain from this?
+
+This section is important as the motivation or problem statement is indepenent from the proposed change. Even if this RFC is not accepted this Motivation can be used for alternative solutions.
+
+In the end, please make sure to present a neutral Problem statement, rather than one that motivates a particular solution
+-->
+
+## Detailed Design or Proposal
+
+<!-- This is the bulk of the RFC. Explain the proposal or design in enough detail for the inteded audience to understand. -->
+
+After we [introduced new github issues beta](https://github.com/microsoft/fluentui/pull/22337) for reporting issues and feature request, the experience for both contributors and shield devs was significantly improved. A followup that was discussed is to implement further automation by leveraging Github Actions, This document will describe the approach in better detail.
+
+We would like to automate following steps:
+
+1. project label assignment
+2. package label assignment
+3. assign issue to project board/team
+
+**Pre-requirements:**
+
+- normalization/unification of labels within monorepo for projects
+
+_Current state:_
+
+- 2 labels for v0
+  - https://github.com/microsoft/fluentui/labels?q=northstar
+- 2 labels for v9
+  - https://github.com/microsoft/fluentui/labels/Fluent%20UI%20react-components%20%28v9%29
+  - https://github.com/microsoft/fluentui/labels/Fluent%20UI%20vNext
+- per component labels
+  - same labels mixed for various fluent version https://github.com/microsoft/fluentui/labels/Component%3A%20Dialog
+
+_Expected state:_
+
+- 1 label for v0
+- 1 label for v9
+- label per published package
+- "component labels" can stay as are but should be used solely for v8 (internal react package "sub-packages" https://github.com/microsoft/fluentui/labels/Component%3A%20Dialog)
+
+### 1. project label assignment
+
+> Note:
+>
+> - by "project" we mean one of libraries that exist within monorepo (v0,v8,v9,web-components)
```

Comment:
does this make sense with respect to the current expectations for partners ? We actively encourage partners to use `@fluentui/react-components`. Using package labels instead of component labels seems like we are fighting against our own recommendation to use the suite package

Reply:
not sure I understand the concern? 

` - by "project" we mean one of libraries that exist within monorepo (v0,v8,v9,web-components)`

we do this today - add label based on project , NOT granular package.

probably you're asking about option 2 (`2. package label assignment`)?  

can you pls clarify? ty

Reply:
yes I 'm asking about `packge label assignment` why should we oblige users to select the correct package in our monorepo when the only exposure they have (and we do it intentionally) is the library package ?

Reply:
answered with proposal https://github.com/microsoft/fluentui/pull/24817#discussion_r988988231